recipes/xtea/meta.yaml

```diff
@@ -20,14 +20,16 @@ requirements:
       - bwa
       - minimap2
       - wtdbg
-
+      - numpy
+      - scikit-learn
+      - pandas
 test:
   commands:
     - xtea --version
     - xtea_hg19 --version
-
+    - xtea_long --version
 about:
     home: https://github.com/parklab/xTea
-    license: Non-commercial use only
+    license: AGPL
```

Comment:
can you please use spdx identifiers? e.g. https://spdx.org/licenses/AGPL-3.0-only.html

Reply:
Thank you @bgruening ! I have added the identifier.